### ISSUE TITLE
Monitoring Page - Profit values in coin amount for Today, Yesterday, Last Week incorrect.

### ISSUE THREAD

**Issue by Brillynt** [taniman/profit-trailer#1445]
If you add the values manually or in a spreadsheet they do not match what is displayed on the Monitoring page.

![image](https://user-images.githubusercontent.com/399748/37464696-7f3e1da2-282f-11e8-9102-0dcbdc8e561f.png)

![image](https://user-images.githubusercontent.com/399748/37464715-9a7ba8be-282f-11e8-8704-c9f4a478c4de.png)

**Comment by Brillynt** [comment]
Version is  P T Ver: 1.2.6.24

**Comment by YoJeff** [comment]
So what am I missing here?   
Your totals show:
.008770 vs .008767 which is within 0.03%
.02253889 vs .02253871 which is within 0.0007 %

**Comment by Brillynt** [comment]
Well I would always assume that the math is being done correctly, not off by a small amount.
To me that is a bug. Those small errors add up overtime to give you larger errors.

**Comment by TheBigShort111** [comment]
I agree with Brillynt on this one... Math is math, plain and simple. It would be a valid argument if rounding were taking place, as .008767 rounds to .008770, however that is not the case as .02253871 does not round up to .02253889, it would simply remain .02253871 as obviously the math is being done out to that number of decimal places.

I've always struggled for the math from PT to match the numbers in my spreadsheet... I've often wondered what was causing the discrepancies (regardless how small) - Like I said, math is a pretty straightforward process. I've come to live with the small variations, however I still do not understand where the discrepancies stem from.

**Comment by taniman** [comment]
![image](https://user-images.githubusercontent.com/6751341/37519222-95d4872e-2918-11e8-9729-cfec2e63d8b5.png)

![image](https://user-images.githubusercontent.com/6751341/37519250-ac1c5f5c-2918-11e8-9523-a819fac5df2d.png)

I got them to show correctly. The 'problem' is that frontend is doing the calculation differently from the backend.

The values on the monitoring page are more accurate cause they come directly from the backend.

the profit ETH collumn is being calculated by the frontend and seems using a bit differnt formula so the values are slight off.

[sales-log.xlsx](https://github.com/taniman/profit-trailer/files/1819061/sales-log.xlsx)

I have attached my excel file if you want to use that formula in your excel file too.
Let me talk to frontender to see if we can fix this in a version soon.
